### PR TITLE
Sumo 236063 fix arm template with security

### PR DESCRIPTION
**Pull request by priyansh-sumo**

### CODE REVIEW

File: AppendBlobReader/src/appendblobreader.json [hunk 1]
```diff
@@ -448,49 +440,81 @@
...
+                            "name": "AzureEventHubConnectionString",
+                            "value": "[listkeys(resourceId('Microsoft.EventHub/namespaces/authorizationRules', parameters('namespaces_BlobReaderNamespace_name'),parameters('AuthorizationRules_RootManageSharedAccessKey_EventHub_name')), '2022-10-01-preview').primaryConnectionString]"
+                        },
+                        {
+                            "name": "TaskQueueConnectionString",
+                            "value": "[listkeys(resourceId('Microsoft.ServiceBus/namespaces/authorizationRules', parameters('namespaces_blobreadertaskqueue_name'),parameters('AuthorizationRules_RootManageSharedAccessKey_TaskQueue_name')), '2022-10-01-preview').primaryConnectionString]"
+                        },
+                        {
+                            "name": "WEBSITE_NODE_DEFAULT_VERSION",
+                            "value": "6.11.0"
```
Comment (himanshu219) [inline AppendBlobReader/src/appendblobreader.json:497]: is this correct ? node version should be 18

Comment (himanshu219) [reply AppendBlobReader/src/appendblobreader.json:497]: fix in other templates also if not present in them as well

**Review by himanshu219 — DISMISSED** [review]

**Comment by himanshu219** [comment]
add the file paths in checkov and arm ttk git actions

File: AppendBlobReader/src/appendblobreader.json (hunk 1 above)
Comment (priyansh-sumo) [reply AppendBlobReader/src/appendblobreader.json]: Updated in Append blob

Comment (priyansh-sumo) [reply AppendBlobReader/src/appendblobreader.json]: appendblobreader.json

**Pull request merged**